metricbeat/module/docker/memory/_meta/fields.yml

```diff
@@ -4,6 +4,12 @@
     Memory metrics.
   release: ga
   fields:
+    - name: stats.*
+      type: object
+      object_type: double
```

Comment:
would `long` be better in this case? I see you expect a `uint64` here

Reply:
Oh derp, you're right.